pset-functions.js

```diff
@@ -33,6 +33,20 @@ console.log('Problem 1:')
 
 // Add your code above this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+    const MAX = 21;
+    if(playerCardScore > MAX && dealerCardScore > MAX){
+        return 0;
+    } else {
+        if(playerCardScore >= playerCardScore && dealerCardScore <= MAX || dealerCardScore > MAX){
```

Comment:
@Ashbk32 be careful with your indentation, it's off.

Also, you want to use `if..else if` in this situation, because you are testing against multiple conditions.